src/lit-element.ts

```diff
@@ -85,7 +79,11 @@ export class LitElement extends UpdatingElement {
 
   /** @nocollapse */
   protected static finalize() {
-    super.finalize();
+    if (this !== LitElement) {
```

Comment:
Add a comment about this if statement.

Is it to match the previous `finalized = true` behavior?

Reply:
Yes, done.

Reply:
(Update: ended up wanting the special name in lit-element.ts after all, since it helps optimize the finalization stack, but I just copied the string instead of exporting something somebody might try and depend on)